src/client/sandbox/event/simulator.js

```diff
@@ -600,6 +600,23 @@ export default class EventSimulator {
         return null;
     }
 
+    _dispatchTextEvent (el, text) {
+        const name = browserUtils.isIE ? 'textinput' : 'textInput';
+
+        let event = null;
```

Comment:
Specification https://webplatform.github.io/docs/dom/TextEvent/initTextEvent.

Rewrite as
```js
 if (nativeMethods.WindowTextEvent && nativeMethods.documentCreateEvent) {
           const event = nativeMethods.documentCreateEvent.call(document, 'TextEvent');         
           const textEventArgs = {
                 eventType: browserUtils.isIE ? 'textinput' : 'textInput',
                 ...other parameters
           };
          
           event.initTextEvent(textEventArgs.eventType, ....);

           return this._raiseDispatchEvent(el, event);
 }

return null;
```

Also there are few questions about ` event.initTextEvent(name, true, true, null, text, 0, '')` parameters:
* `null` specified for  window on which this event is occurring. Maybe it should be the current window instead of null?
* 0 specified how text is entered. Now it equals `TextEvent.DOM_INPUT_METHOD_UNKNOWN = 0`. Maybe we should use `TextEvent.DOM_INPUT_METHOD_KEYBOARD = 1`?
* '' specified the locale name. As I understand it should be the current locale name - `navigator.language`.

Reply:
I agree